templates/todo/projects/nodejs-mongo/repo-bicep.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   description: ToDo Application with a Node.js API and Azure Cosmos DB API for MongoDB
 
 repo:
```

Comment:
Are you thinking that there would be a repo-terraform.yaml?

Reply:
in the proposed design, Every projec within the todo folder can have multiple azure-<IACTOOL>.yml, and repo-<IACTOOL>.yml. along with a sub folder by IAC name within infra . 

Each version of the repo-<IAC>.yaml will define the corresponing remote repo name, and URL. along with the content of the infra folder as well.

 This give a path to adding the multi IAC tool support and deploying to different iac sample templates with out duplicating projects within todo folder 
![image](https://user-images.githubusercontent.com/34171319/185668845-8ded94fc-fdd1-4511-a008-2e34f02c0a40.png)